onedal/neighbors/neighbors.py

```diff
@@ -549,7 +552,8 @@ def _onedal_fit(self, X, y, queue):
 
             return train_alg(**params).compute(X, y).model
 
-        policy = _get_policy(queue, X, y)
+        policy = self._get_policy(queue, X, y)
+        X, y = _convert_to_supported(policy, X, y)
```

Comment:
Is there any performance measurement to prove that this code didn't affect performance especially for small data shapes cases?

Reply:
The inclusion or exclusion of this line does not impact performance (runtime, results) for examples of various sizes from a few basic experiments I have tried. @KulikovNikita have you observed improvements?